storage/store.go

```diff
@@ -1602,10 +1602,7 @@ func (s *Store) processRaft() {
 							groupID, cmd)
 						log.Error(err)
 					} else {
-						err = r.processRaftCommand(cmdIDKey(commandID), index, cmd)
-					}
-					if callback != nil {
-						callback(err)
+						r.addCmdQueue(cmdIDKey(commandID), index, cmd, callback)
```

Comment:
we already have a "command queue" so naming this `addCmdQueue` could be confusing. I'd prefer something like `applyQueue`.